package.json

```diff
@@ -1,75 +1,86 @@
-    "express": "^4.16.4",
-    "ext2fs": "^1.0.26",
-    "firmata": "^2.0.0",
-    "lodash": "^4.17.11",
-    "mountutils": "^1.3.16",
-    "mz": "^2.7.0",
-    "resin-cli-visuals": "^1.4.2",
-    "resin-image-fs": "^5.0.6"
-  },
-  "devDependencies": {
-    "@types/bluebird-retry": "^0.11.2",
-    "@types/chai": "^4.1.7",
-    "@types/chai-as-promised": "^7.1.0",
-    "@types/chai-http": "^3.0.5",
-    "@types/express": "^4.16.1",
-    "@types/firmata": "^0.19.0",
-    "@types/lodash": "^4.14.120",
-    "@types/mocha": "^5.2.5",
-    "@types/mz": "0.0.32",
-    "@types/node": "^10.12.18",
-    "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
-    "chai-http": "^4.2.1",
-    "husky": "^1.3.1",
-    "lint-staged": "^8.1.0",
-    "mocha": "^5.2.0",
-    "nyc": "^13.2.0",
-    "prettier": "^1.16.1",
-    "resin-lint": "^2.0.1",
-    "sinon": "^7.2.3",
-    "ts-mock-imports": "^1.2.2",
-    "typescript": "^3.2.4"
-  }
+	"name": "leviathan-worker",
+	"version": "0.0.2",
+	"description": "RESTful API to send commands to a testbot",
+	"main": "build/index.js",
+	"directories": {
+		"lib": "lib",
+		"bin": "bin",
+		"test": "test"
+	},
+	"scripts": {
+		"clean": "rm -rf build",
+		"prettify": "prettier --config ./node_modules/resin-lint/config/.prettierrc --write \"{lib,test}/**/*.ts\"",
+		"test": "npm run prettify && npm run lint && npm run test:cov",
+		"lint": "resin-lint --typescript lib/ test/ && tsc --noEmit",
+		"build": "npm run clean && npm run prettify && tsc",
+		"build:test": "npm run clean && tsc --project . && cp -r test/assets/image.zip build/test",
+		"start": "node ./build/bin",
+		"test:cov": "npm run build:test && nyc mocha",
+		"test:fast": "npm run build:test && mocha"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/balena-io/leviathan-worker.git"
+	},
+	"keywords": [
+		"testbot",
+		"leviathan",
+		"worker"
+	],
+	"author": "Theodor Gherzan",
+	"license": "Apache-2.0",
+	"bugs": {
+		"url": "https://github.com/balena-io/leviathan-worker/issues"
+	},
+	"homepage": "https://github.com/balena-io/leviathan-worker#readme",
+	"dependencies": {
+		"@types/bluebird": "^3.5.25",
+		"async-mutex": "^0.1.3",
+		"bluebird": "^3.5.3",
+		"bluebird-retry": "^0.11.0",
+		"etcher-sdk": "^1.3.1",
+		"express": "^4.16.4",
+		"ext2fs": "^1.0.26",
+		"firmata": "^2.0.0",
+		"lodash": "^4.17.11",
+		"mountutils": "^1.3.16",
+		"mz": "^2.7.0",
+		"resin-cli-visuals": "^1.4.2",
+		"resin-image-fs": "^5.0.6"
+	},
+	"devDependencies": {
+		"@types/bluebird-retry": "^0.11.2",
+		"@types/chai": "^4.1.7",
+		"@types/chai-as-promised": "^7.1.0",
+		"@types/chai-http": "^3.0.5",
+		"@types/express": "^4.16.1",
+		"@types/firmata": "^0.19.0",
+		"@types/lodash": "^4.14.120",
+		"@types/mocha": "^5.2.5",
+		"@types/mz": "0.0.32",
+		"@types/node": "^10.12.18",
+		"@types/request-promise": "^4.1.42",
+		"chai": "^4.2.0",
+		"chai-as-promised": "^7.1.1",
+		"chai-http": "^4.2.1",
+		"drivelist": "^8.0.2",
```

Comment:
This is required by etcher-sdk, I don't think you need it in package.json.

Reply:
I need this for mocking.

Reply:
It should be already available by installing `etcher-sdk`. I wouldn't put it in `package.json` because you might end up installing a different version that `etcher-sdk` uses and mock the wrong code.